tests/source/arrow_in_comments/arrow_in_single_comment.rs

```diff
@@ -1,4 +1,4 @@
-// rustfmt-version: Two
+// rustfmt-style_edition: 2024
```

Comment:
I opted to do these as direct mappings, version Two become edition 2024 and version One edition 2015.

There's a reasonable case to be made that we should fan these out across all style editions, but (1) I don't think we should do that in this PR and (2) if we go that route I think we need to make that standard practice and apply it consistently moving forward as well